crates/router/src/core/payments/flows.rs

```diff
@@ -142,6 +142,7 @@ impl<const T: u8>
 }
 
 default_imp_for_complete_authorize!(
+    connector::Gocardless,
```

Comment:
Please maintain the order here as well as in the other default implementations.

Reply:
@prasunna09 I resolved it already a separate PR -https://github.com/juspay/hyperswitch/pull/2180